jobs/config.json

```diff
@@ -9315,14 +9315,13 @@
       "--env-file=jobs/platform/gce.env",
       "--env-file=jobs/env/ci-kubernetes-kubemark-5-gce.env",
       "--extract=ci/latest",
-      "--extract-source",
-      "--gcp-project=sen-lu-test",
+      "--gcp-project=k8s-jenkins-kubemark",
```

Comment:
Also I think there is `--gcp-node-image=gci` we're passing to the kubemark jobs.

Reply:
That might be the reason why those runs are failing during the ssh step where we're running some setup scripts (which iirc have gci-specific stuff that don't work on container_vm)

Reply:
that should be default now

Reply:
just in case, I added `--gcp-node-image=gci`

Reply:
I see. We probably want to get rid of the flag for the jobs then (though it can happen in a separate PR).